app/views/videos/show.html.haml

```diff
@@ -8,18 +8,23 @@
           %h3= @video.title
         %p= @video.description
         .actions
-          %a{:class => "btn btn-primary"} Watch Now
-          %a{:class => "btn btn-default"} + My Queue
+          = link_to "+ My Queue", queue_items_path(video_id: @video.id), method: :post, class: "btn btn-default"
 
 %section.reviews.container
   .row
     .col-sm-10.col-sm-offset-1
       = form_for [@video, Review.new] do |f|
         %fieldset
-          %label Rate this video
-          = f.select :rating, options_for_select([5,4,3,2,1].map { |number| [pluralize(number, "Star")]})
-          %label Write Review
-          = f.text_area :content, rows: 6, class: "span6"
+          .form_group
+            %label Rate this video
+            .row
+              .col-sm-3
+                = f.select :rating, options_for_select([5,4,3,2,1].map { |number| [pluralize(number, "Star")]})
```

Comment:
You should pass a number to the array building options for select menu. You want to pass the number as ID not the string.